WordPress/gutenberg
# Tabs: Auto-generate README

Stacked on #68208

## What?

Convert the Tabs README to an auto-generated one.

Supplementary information that was in the existing README is moved to other appropriate locations (JSDocs, Storybook "Best Practices" page).

## Why?

To decrease maintenance cost and consolidate the canonical docs for our audience.

## Testing Instructions

- In your IDE, check imports of `Tabs.Tab` for example, and see that the JSDoc includes the subcomponent description.
- `npm run docs:components` to regenerate READMEs.
- See Storybook docs for Tabs.
